SharpPcap/LibPcap/LibPcapSafeNativeMethods.Windows.cs

```diff
@@ -0,0 +1,266 @@
+/*
+This file is part of SharpPcap.
+
+SharpPcap is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+SharpPcap is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with SharpPcap.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/* 
+ * Copyright 2005 Tamir Gal <tamir@tamirgal.com>
+ * Copyright 2008-2009 Phillip Lemon <lucidcomms@gmail.com>
+ * Copyright 2008-2011 Chris Morgan <chmorgan@gmail.com>
+ */
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Text;
+
+namespace SharpPcap.LibPcap
+{
+    /// <summary>
+    /// Per http://msdn.microsoft.com/en-us/ms182161.aspx 
+    /// </summary>
+    [SuppressUnmanagedCodeSecurityAttribute]   
+    internal static class Windows
+    {
+        // NOTE: For mono users on non-windows platforms a .config file is used to map
```

Comment:
Should we remove this note now that this file is windows specific?

Reply:
Removed.